src/components/TelematicReceiptFlowExportOverview/TelematicReceiptFlowExportOverview.tsx

```diff
@@ -0,0 +1,112 @@
+                {t('telematicReceiptFlowExportOverview.description')}
+            </Typography>
+            <Grid container direction="row" spacing={2}
+                sx={{
+                    alignItems: 'center',
+                    justifyContent: 'space-between',
+                    marginBottom: 2
+                }}>
+                <Grid item lg={5}>
+                    <TextField
+                        sx={{ bgcolor: theme.palette.common.white }}
+                        fullWidth
+                        size="small"
+                        InputProps={{
+                            startAdornment: <InputAdornment position="start"><Search /></InputAdornment>
+                        }}
+                        label={t('telematicReceiptFlowExportOverview.searchFlowName')}
+                    />
+                </Grid>
+                <Grid item lg={2}>
+                    <TextField
+                        sx={{ bgcolor: theme.palette.common.white }}
+                        fullWidth
+                        size="small"
+                        InputProps={{
+                            endAdornment: <InputAdornment position="end"><CalendarToday /></InputAdornment>
+                        }}
+                        label={t('telematicReceiptFlowExportOverview.from')}
+                    />
+                </Grid>
+                <Grid item lg={2}>
+                    <TextField
+                        sx={{ bgcolor: theme.palette.common.white }}
+                        fullWidth
+                        size="small"
+                        InputProps={{
+                            endAdornment: <InputAdornment position="end"><CalendarToday /></InputAdornment>
+                        }}
+                        label={t('telematicReceiptFlowExportOverview.to')}
+                    />
+                </Grid>
+                <Grid item lg={1}>
+                    <Button
+                        fullWidth
+                        size="medium"
+                        variant="contained"
+                        sx={{height: 40}}>
+                        {t('telematicReceiptFlowExportOverview.filter')}
+                    </Button>
+                </Grid>
+                <Grid item lg={2}
+                    display={'flex'}
+                    justifyContent={'center'}
+                    alignContent={'center'} 
+                >
+                    <ButtonNaked
+                        fullWidth
+                        color='text'
+                        weight='default'
+                        size='small'
+                        onFocusVisible={function noRefCheck() { }}
+                    >
+                        {t('telematicReceiptFlowExportOverview.removeFilter')}
+                    </ButtonNaked>
+                </Grid>
+            </Grid>
+            <Box
+                sx={{
+                    borderColor: theme.palette.grey[200],
+                    borderWidth: 20,
```

Comment:
Can you try to obtain the result in a different way? A border of 20pixels is really cringe